cloud.conf.json.template

```diff
@@ -0,0 +1,12 @@
+{
+    "default": {
```

Comment:
You can name your clouds any way you like. For multiple ports just add multiple cloud entries with the same host but different ports.